api/queue/queue.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2023 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package queue
+
+import (
+	"github.com/go-vela/server/router/middleware/claims"
+	"net/http"
+
+	"github.com/go-vela/types/library"
+
+	"github.com/gin-gonic/gin"
+	"github.com/sirupsen/logrus"
+)
+
+// swagger:operation POST /api/v1/queue/queue-registration admin QueueRegistration
```

Comment:
```suggestion
// swagger:operation POST /api/v1/queue/register queue Registration
```

Reply:
also personal preference, related to my other comment:
```suggestion
// swagger:operation POST /api/v1/queue/queue-registration queue Register
```